web/src/javascript/gameUI.js

```diff
@@ -36,24 +36,30 @@ export function resizeCanvas(canvas, width, height) {
   return resized;
 }
 
-export function displayLoading() {
-  let state = "Loading ";
-  return setInterval(() => {
-    const screenCanvas = document.getElementById("screenCanvas");
-    const context = screenCanvas.getContext("2d");
-    const screenWidth = screenCanvas.width;
-    const screenHeight = screenCanvas.height;
+function print(str) {
+  const screenCanvas = document.getElementById("screenCanvas");
+  const context = screenCanvas.getContext("2d");
+  const screenWidth = screenCanvas.width;
+  const screenHeight = screenCanvas.height;
 
-    context.clearRect(0, 0, screenWidth, screenHeight);
-    context.beginPath();
-    context.fillStyle = constants.textBorderColor;
-    context.font = constants.textStyle;
-    context.textAlign="center";
-    context.textBaseline = "middle";
-    context.lineWidth = constants.textBorderThickness;
+  context.clearRect(0, 0, screenWidth, screenHeight);
+  context.rect(0, 0, screenWidth, screenHeight);
+  context.fillStyle = "#ffffff";
+  context.fill();
+  context.beginPath();
+  context.fillStyle = constants.textBorderColor;
```

Comment:
Not sure but it looks like you are already assigning `context.fillStyle` 3 lines before